html/janus.js

```diff
@@ -1210,6 +1210,12 @@ function Janus(gatewayCallbacks) {
 			var constraints = { mandatory: {}, optional: []};
 			pluginHandle.consentDialog(true);
 			var videoSupport = isVideoSendEnabled(media);
+			var audioSupport = isAudioSendEnabled(media);
+			if(audioSupport === true && media != undefined && media != null) {
+				if(typeof media.audio === 'object') {
+					audioSupport = media.audio;
```

Comment:
Ah ok, got it, thanks. So this partly answers one of the questions I made below.